docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,778 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    def _foreward_processing(
</pre><p>Let's shorten this to <code>_foreward_process</code> and <code>_backward_process</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Will revise the name of these two functions.</p>